q2cli/core/usage.py

```diff
@@ -134,11 +134,11 @@ def init_artifact(self, name, factory):
         return variable
 
     def import_from_format(self, name, semantic_type,
-                           fmt_variable, view_type=None):
-        variable = super().import_from_format(
-            name, semantic_type, fmt_variable, view_type=view_type)
+                           variable, view_type=None):
+        var = super().import_from_format(
+            name, semantic_type, variable, view_type=view_type)
 
-        in_fp = fmt_variable.to_interface_name()
+        in_fp = variable.to_interface_name()
         out_fp = variable.to_interface_name()
```

Comment:
This has to be changed to `var` (hopefully answering the question for why there are two). You can see these used on lines 153-4

Reply:
Sorry for all the derping around with this @ebolyen  and @thermokarst  - I've been making bad inferences from intermediate states when I should have been reading the original code. Happy New Year! :tada: